Frontend/ui-customer/src/Services/API.purs

```diff
@@ -900,6 +900,7 @@ newtype RideBookingRes = RideBookingRes {
   tripTerms :: Array String,
   id :: String,
   updatedAt :: String,
+  hasNightIssue :: Boolean,
```

Comment:
@PraveenGongada are the changes from backend deployed?

Reply:
Not yet